src/DotNetWorker.Core/FunctionsApplication.cs

```diff
@@ -81,6 +81,8 @@ public async Task InvokeFunctionAsync(FunctionContext context)
                 invokeActivity?.SetStatus(ActivityStatusCode.Error, ex.Message);
                 invokeActivity?.RecordException(ex, escaped: true);
```

Comment:
I think part of the offline conversation with @lmolkova recommended removing this?

> so there is no valid reason to use RecordException if you have ILogger

If you don't want to remove it just yet / have further discussion that is fine - can leave that work separate to this PR.

Reply:
Yep -- removed it in an updated commit and added some Activity-only tests as well (previous tests were all App Insights-based)